src/pages/instances/actions/ConvertToInstanceBtn.tsx

```diff
@@ -0,0 +1,321 @@
+            buttonRow={
+              <>
+                <Button
+                  appearance="base"
+                  className="u-no-margin--bottom"
+                  type="button"
+                  onClick={handleCloseModal}
+                >
+                  Cancel
+                </Button>
+                <ActionButton
+                  appearance="positive"
+                  className="u-no-margin--bottom"
+                  loading={formik.isSubmitting || !!uploadState}
+                  disabled={
+                    !formik.isValid ||
+                    isProjectLoading ||
+                    !formik.values.imageFile
+                  }
+                  onClick={() => void formik.submitForm()}
+                >
+                  Convert and create
+                </ActionButton>
+              </>
+            }
+          >
+            <Form
+              onSubmit={formik.handleSubmit}
+              className={uploadState ? "u-hide" : ""}
+            >
+              <Input
+                id="image-file"
+                name="imageFile"
+                type="file"
+                label="Image file"
+                onChange={handleFileChange}
+              />
+              <Input
+                {...formik.getFieldProps("name")}
+                id="name"
+                type="text"
+                label="New instance name"
+                placeholder="Enter name"
+                error={formik.touched.name ? formik.errors.name : null}
+                disabled={!formik.values.imageFile}
+              />
+              <StoragePoolSelector
+                project={project?.name || ""}
+                value={formik.values.pool}
+                setValue={(value) => void formik.setFieldValue("pool", value)}
+                selectProps={{
+                  id: "pool",
+                  label: "Root storage pool",
+                  disabled: isProjectLoading || !formik.values.imageFile,
+                }}
+              />
+              <Select
+                {...formik.getFieldProps("architecture")}
+                id="architecture"
+                label="Image architecture"
+                options={architectureOptions}
+                disabled={!formik.values.imageFile}
+              />
+              <label htmlFor="">Conversion options</label>
+              <div className="conversion-options">
+                <Input
+                  {...formik.getFieldProps("formatConversion")}
+                  type="checkbox"
+                  label="Format"
+                  disabled={!formik.values.imageFile}
+                  checked={formik.values.formatConversion}
+                />
+                <Input
+                  {...formik.getFieldProps("virtioConversion")}
+                  type="checkbox"
+                  label="Virtio"
+                  disabled={!formik.values.imageFile}
+                  checked={formik.values.virtioConversion}
+                />
```

Comment:
I think we need some help text here, as these are not self-explanatory to a user.

Reply:
I've added some help text for both both options, let me know what you think.